com.unity.shadergraph/Editor/Data/Graphs/GraphData.cs

```diff
@@ -2516,13 +2516,13 @@ internal void ValidateCustomBlockLimit()
             {
                 ClearErrorsForNode(cib);
                 total += (int)cib.customWidth;
-                if (total > d3dSupport)
+                if (total > errorLevel)
                 {
-                    AddValidationError(cib.objectId, $"{cib.customName} may exceed interpolation channel limitations on most platforms (such as Direct3D).");
+                    AddValidationError(cib.objectId, $"{cib.customName} exceeds the interpolation channel error threshold: {errRange}. See ShaderGraph project settings.");
                 }
-                else if (total > chromeSupport)
+                else if (total > warnLevel)
                 {
-                    AddValidationError(cib.objectId, $"{cib.customName} may exceed interpolation channel limitations on low-end platforms (such as Chrome's WebGL).", ShaderCompilerMessageSeverity.Warning);
+                    AddValidationError(cib.objectId, $"{cib.customName} exceeds the interpolation channel warning threshold: {warnRange}. See ShaderGraph project settings.", ShaderCompilerMessageSeverity.Warning);
```

Comment:
@ValGrimm-U3D These are the warning/error messages.

Reply:
Thanks! I'd remove the reference "(such as Chrome's WebGL)" as a specific platform reference is not needed, and WebGL is not specific to Chrome. Otherwise looks ok.

Reply:
Oh wait, I see you've already done this.